internal/cmd/skupper/system/nonkube/system_delete.go

```diff
@@ -123,8 +124,10 @@ func (cmd *CmdSystemDelete) Run() error {
 			err := cmd.siteHandler.Delete(site.Name)
 			if err != nil {
 				slog.Error("Error while deleting site %q: %s", site.Name, err)
```

Comment:
The given format should not be passed directly to slog.Error.
I guess you'd need to use `fmt.Sprintf` along with it.

Reply:
Nice catch. godocs with context on what that may do- (looks it'd combine the args into one Arg {{ site.Name}}={{ err.Error() }}: https://pkg.go.dev/log/slog#Logger.Log

We haven't really formalized structured logging style at all, but I may do something like this:
`slog.Error("Error while deleting site", slog.String("site_name", site.Name), slog.Any("error", err))`